enterprise-suite/templates/prometheus-deployment.yaml

```diff
@@ -90,8 +90,12 @@ spec:
           args:
             - --configPath=/etc/config/
             - --storagePath=/monitor-data/
-            - --staticConfig=/etc/console-api/static-rules.yml
-            - --defaultMonitorsFile=/etc/console-api/default-monitors.json
+            {{ if .Values.defaultMonitorsFile }}
+            - --defaultMonitorsFile=/etc/{{ .Values.defaultMonitorsFile }}
+            {{ end }}
+            {{ if .Values.staticRulesFile }}
+            - --staticRules=/etc/{{ .Values.staticRulesFile }}
```

Comment:
I presume there's a corresponding change to console-api to use `--staticRules` instead of `--staticConfig`.

Reply:
Oh, that's just a mistype. Changing to `--staticConfig`.